comprobo_final_project/scripts/simulator/robot.py

```diff
@@ -46,10 +55,11 @@ def step(self, step_size):
 
         # Update velocity
         self.pose.velocity.x = twist_r * math.cos(original_velocity_theta +
-                step_size * twist_theta)
+                twist_theta / step_freq)
```

Comment:
Checked, it works. Just shifting the code to follow the same standard. Our reference frame for everything should now be in frequencies (Hertz)